app/javascript/gobierto_visualizations/webapp/visualizations/treeMapNested.vue

```diff
@@ -168,14 +174,24 @@ export default {
       const dataTreeMap = JSON.parse(JSON.stringify(data));
       this.transformDataTreemap(dataTreeMap)
     },
+    createScaleColors(values) {
+      let colorsGobiertoExtend = ["#12365b", "#118e9c", "#ff766c", "#f7b200", "#158a2c", "#94d2cf", "#3a78c3", "#15dec5", "#6a7f2f", "#55f17b"]
+      colorsGobiertoExtend = colorsGobiertoExtend.slice(0, values)
+
+      const colors = d3.scaleOrdinal()
+        .domain(this.arrayValuesContractTypes)
+        .range(colorsGobiertoExtend);
+      return colors;
+    },
```

Comment:
`<mode ironic="on">` Oh, nice, two times the same function 🎉  `<mode ironic="off">`

Reply:
Done!